recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
     - {{ compiler('cxx') }}
     - make  # [unix]
     - cmake >=3.10
+    - lldb
```

Comment:
huh, missing?
Isn't it here: https://anaconda.org/conda-forge/lldb ?

Reply:
depends on https://github.com/conda-forge/lldb-feedstock/pull/8 depends on https://github.com/conda-forge/lldb-feedstock/pull/5